google/cloud/spanner_dbapi/connection.py

```diff
@@ -4,160 +4,129 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-"""Cloud Spanner DB connection object."""
+"""DB-API Connection for the Google Cloud Spanner."""
 
-from collections import namedtuple
-import warnings
+from google.api_core.gapic_v1.client_info import ClientInfo
+from google.cloud import spanner_v1 as spanner
 
-from google.cloud import spanner_v1
+from google.cloud.spanner_dbapi.cursor import Cursor
+from google.cloud.spanner_dbapi.exceptions import InterfaceError
+from google.cloud.spanner_dbapi.version import DEFAULT_USER_AGENT
+from google.cloud.spanner_dbapi.version import PY_VERSION
 
-from .cursor import Cursor
-from .exceptions import InterfaceError
 
-AUTOCOMMIT_MODE_WARNING = (
-    "This method is non-operational, as Cloud Spanner"
-    "DB API always works in `autocommit` mode."
-    "See https://github.com/googleapis/python-spanner-django#transaction-management-isnt-supported"
-)
```

Comment:
I don't see where this constant will be. We still need it for `autcoommit` mode

Reply:
This is unused, hence was removed from this version. It can be added back in later commits as needed.